composer.json

```diff
@@ -26,8 +26,8 @@
     "require": {
         "php": ">=7.0.0",
         "cloudinary/cloudinary_php": "^2.4",
-        "ozdemirburak/iris": "^2.4",
-        "silverstripe/framework": "^4.6.0"
+        "ozdemirburak/iris": "^2.0",
+        "silverstripe/framework": "^4.5"
```

Comment:
You downgraded these?
Any reason why?

Reply:
Supported LA Phil, plus it also allows us to migrate this to older SS4 clients which were using the not so good version of this module.